src/index.html

```diff
@@ -1,19 +1,590 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Title</title>
+
+    <link
+      rel="icon"
+      type="image/png"
+      href="./public/images/favicon.png"
+    />
+
+    <title>Мистецтво ХІХ - ХХ ст</title>
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="./styles/style.css"
     />
   </head>
   <body>
-    <h1>Hello Mate Academy</h1>
-    <script src="scripts/main.js"></script>
+    <!-- HEADER -->
+    <header class="header">
+      <div class="section__wrapper header__wrapper">
+        <div class="header__top">
+          <a
+            href="#"
+            class="header__logo icon icon-logo"
+          ></a>
+
+          <div class="header__icons">
+            <a
+              href="#menu"
+              class="icon header__icon icon-menu"
+            ></a>
+            <button class="header__switch-language">
+              UA
+              <span class="icon header__icon-dropdown icon-dropdown"></span>
+            </button>
+          </div>
+        </div>
+
+        <div class="header__bottom">
+          <p class="header__date typography--additional">
+            10 серпня - 10 листопада
+          </p>
+          <h1 class="header__title">Мистецтво ХІХ - ХХ ст.</h1>
+
+          <p class="header__description">
+            Внесок українських митців у світову культуру 19-20 ст.
+          </p>
+
+          <button class="button header__button">Купити квиток</button>
```

Comment:
Maybe let's make all the buttons using `a` and move to the form on click (by ID)?
In this way, u will make your page more dynamic

**Up to you**